opm-core/Installation.rst

```diff
@@ -157,35 +153,29 @@ The dispatcher itself::
 
 .. note::
 
-    With our previous examples, **db_user** would've been set to ``user1`` and
-    **db_password** should be set to ``password1``.
+    With our previous examples, **db_user** would've been set to ``opm`` and
+    **db_password** should be set to ``anothergoodpassword``. If you use Icinga2,
+    ``directory`` must be set to ``/var/spool/icinga2/perfdata/``.
```

Comment:
You should say that this directory is default one (unless of course it can't be changed on incinga2)

Reply:
Done.